src/ReduxRouter.js

```diff
@@ -91,7 +91,7 @@ class ReduxRouter extends Component {
 class ReduxRouterContext extends Component {
   static propTypes = {
     location: PropTypes.object,
-    RoutingContext: PropTypes.element,
+    RoutingContext: PropTypes.instanceOf(Component)
```

Comment:
@Scarysize I checked both variants. And they seem to work. You may choose whichever you like more.


```js
PropTypes.instanceOf(Component)
```

```js
PropTypes.oneOfType([
   PropTypes.instanceOf(Component),
   PropTypes.func
])
```